src/screens/Profile/Login/PhoneInput.tsx

```diff
@@ -0,0 +1,68 @@
+import FullScreenHeader from '@atb/components/screen-header/full-header';
+import {StyleSheet} from '@atb/theme';
+import {useTranslation} from '@atb/translations';
+import React, {useState} from 'react';
+import {ActivityIndicator, ScrollView, View} from 'react-native';
+import {LoginNavigationProp} from './';
+import * as Sections from '@atb/components/sections';
+import Button from '@atb/components/button';
+import {useAuthState} from '@atb/auth';
+import ThemeText from '@atb/components/text';
+import ThemeIcon from '@atb/components/theme-icon/theme-icon';
+import {ArrowRight} from '@atb/assets/svg/icons/navigation';
+
+export type PhoneInputProps = {
+  navigation: LoginNavigationProp;
+};
+
+export default function PhoneInput({navigation}: PhoneInputProps) {
+  const {t} = useTranslation();
+  const styles = useThemeStyles();
+  const {signInWithPhoneNumber} = useAuthState();
+  const [phoneNumber, setPhoneNumber] = useState('');
+
+  const onNext = async () => {
+    await signInWithPhoneNumber(phoneNumber);
```

Comment:
Might want to trigger a loading event/indicator before this. It takes a bit of time to navigate to the next screen.

Reply:
Fixed

Reply:
Maybe check out `useIsLoading` (in `@atb/utils/use-is-loading.ts`)